src/Modules/Backend/config/services.yaml

```diff
@@ -28,3 +28,6 @@ services:
 
   ForkCMS\Modules\Backend\Console\:
     resource: '../Console/*'
+
+  ForkCMS\Modules\Backend\Domain\Authentication\TwoFactorAuthenticationFactory:
+    public: true
```

Comment:
empty line at the end is missing

Reply:
fixed 0800d541db23b391af25d1d4d12fa80efc4a3c59